### PR TITLE
Add travis deployment for xDai

### PR DESCRIPTION
**Pull request by dasanra**
Add instruction for apps deployment on xDai network

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -45,7 +50,9 @@ deploy:
     local_dir: build
     region: $AWS_REGION
     on:
-      branch: master
+      # branch: master
+      all_branches: true
+      condition: $TRAVIS_BRANCH =~ ^(master|feature/xdai-deployment)$
```
Comment (rmeissner) [inline .travis.yml:55]: here `master` is on the same level as `feature/xdai-deployment` ... in this case I would recommend not to call it `feature/...` (maybe just `xdai-deployment` or `release/...` not sure). Feature branches are for me used when developing a feature.

Comment (nicosampler) [reply .travis.yml:55]: I think it's just temporal, then all the environments will be deployed together when we merge to master.

Comment (nicosampler) [reply .travis.yml]: @rmeissner updated. I think `xdai-deployment` it's self explanatory

Comment (dasanra) [reply .travis.yml]: Yes, this was a special case to make it deploy to staging without merging to master. I think that this can be removed now

**Review by nicosampler — APPROVED** [review]

**Pull request merged**